bedrock/mozorg/templatetags/misc.py

```diff
@@ -630,13 +630,18 @@ def play_store_url(ctx, product, campaign=None):
 
 @library.global_function
 @jinja2.pass_context
-def ms_store_url(ctx, product, mode="direct", campaign=None):
+def ms_store_url(ctx, product="firefox", mode="mini", campaign=None, handler=None):
     """
     Returns a Microsoft Windows Store URL for a given product.
     Installer mode parameter options include "direct" or "full", or "mini".
     See https://apps.microsoft.com/badge for details.
     """
-    base_url = getattr(settings, f"MICROSOFT_WINDOWS_STORE_{product.upper()}_LINK")
+
+    if handler == "ms-windows-store":
+        base_url = getattr(settings, f"MICROSOFT_WINDOWS_STORE_{product.upper()}_DIRECT_LINK")
+    else:
+        base_url = getattr(settings, f"MICROSOFT_WINDOWS_STORE_{product.upper()}_WEB_LINK")
```

Comment:
I expect this is to stay, not just part of the experiment. Do you plan to document the helper (like the other store links) and this change too in the .rst some day? That said, it has some defaults, and here for the experiment you chose to use `ms-windows-store:` instead of `https:` ~ which one is generally better and what should normally be used when making use of the helper? (If there are [dis]advantages or reasons to prefer one handler to the other, whether that's something to end up described in the docs for posterity.)

Reply:
Yes, we can add docs for this after the experiment concludes 👍